package.json

```diff
@@ -23,7 +23,6 @@
   ],
   "type": "module",
   "exports": {
```

Comment:
I guess now we can expose types for GL Draw as we do in GL JS https://github.com/mapbox/mapbox-gl-js/blob/main/package.json#L130